.github/workflows/scheduled_aas_deploy.yml

```
@@ -0,0 +1,101 @@
+name: Scheduled AAS Deployment
+
+on:
+  schedule:
+    - cron: '30 21 * * FRI'  # Every Friday at 9:30 PM UTC
```

Theoretically, someone could merge _just_ before the code freeze. Given we need to wait for all the release artifacts to be ready, which can take ~1hr in many cases, it may be safest to push this back an hour? It probably doesn't make much difference overall, and may reduce number of failures?
```suggestion
    - cron: '30 22 * * FRI'  # Every Friday at 10:30 PM UTC
```

---

Ahh yeah

That seems fine to me